examples/ensemble/plot_stack_predictors.py

```diff
@@ -249,11 +252,11 @@ def plot_regression_results(ax, y_true, y_pred, title, scores, elapsed_time):
 ):
     start_time = time.time()
     score = cross_validate(
-        est, X, y, scoring=["r2", "neg_mean_absolute_error"], n_jobs=2, verbose=0
+        est, X, y, scoring=["r2", "neg_mean_absolute_error"], n_jobs=2, verbose=0, cv=2
```

Comment:
I think setting `cv=2` `cross_validate` would promote bad ML practices. In this case, I prefer to have a longer running example with the default `cv=5`.

Reply:
That is fine with me, it still would cut the execution time to about the half, so it would anyway be a win.

Reply:
I pushed a commit that removes the above cv=2 part.